app/shared/notifications/_notifications.scss

```diff
@@ -1,13 +1,28 @@
 .notification-container {
+  .notification {
+    border-radius: 0;
+    color: $white;
+    opacity: 1;
+  }
   .notification-info {
     background-color: $brand-info;
+    box-shadow: 0 0 6px $brand-info;
```

Comment:
Is there a reason for 6px?

Reply:
Not really, just prettier than the default (`0 0 12px #999`) that needed to change.